src/app/malaria-101/activity-2/activity-2.component.ts

```diff
@@ -17,10 +17,17 @@ export class DragdropComponent implements OnInit {
     public dropCheckDo: String;
     public dropCheckDont: String;
     language: any;
-    private score = 0;
+    public score = 0;
```

Comment:
it is a good practice to keep it private and write a public method say getScore, which will return the score value when ever you call.